.github/workflows/build.yaml

```diff
@@ -43,3 +45,29 @@ jobs:
             type: localBlob
             localReference: extension_definitions.tar.gz
         blobs-directory: blobs.d
+
+  helmchart-imagemap:
+    runs-on: ubuntu-latest
+    needs:
+      - prepare
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Export Helmchart-Imagemap
+        run: |
```

Comment:
not sure whether it is the default; I think setting `set -eu [o pipefail]` is a good practice